shared/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "",
   "dependencies": {
+    "bson-ext": "^4.0.2",
```

Comment:
I don't think we need this package in shared, that is more of a backend dependency. Should be mindful of what is truly needed